docs/src/pages/index.mdx

```diff
@@ -0,0 +1,178 @@
+---
+layout: ../layouts/wiki.astro
+---
+
+import InstallCommands from "@/components/install-commands.astro";
+
+# Remeda
+
+Fully written in TypeScript for powerful, type-safe, and functional utilities.
+
+<InstallCommands />
+
+## 🧰 Expansive
+
+Remeda offers a wide range of common everyday utility functions so that you
+don't have to deal with the minutiae of writing them yourself. We ensure that
+our algorithms are efficient, optimized, and cover all edge cases properly. We
+offer a wide range of functions, from simple utilities like
+[`partition`](/docs#partition) and building objects with
+[`pullObject`](/docs#pullObject), to _complex_ algorithms like
+[`takeFirstBy`](/docs#takeFirstBy).
+
+```ts
+type User = {
```

Comment:
**nit:** let's add the import to show usage

```suggestion
```ts
import * as R from "remeda";

type User = {
```